en/integrations/plugin-packs/procedures/virtualization-hpe-simplivity-restapi.md

```diff
@@ -0,0 +1,234 @@
+checking host 'svt01.acme.com'
+    status: alive
+    logical drive '1' status: green
+    logical drive '3' status: green
+    physical drive '1:1-1' status: green
+    physical drive '1:1-2' status: green
+    physical drive '1:1-3' status: green
+    physical drive '1:1-4' status: green
+    physical drive '1:1-5' status: green
+    physical drive '1:1-6' status: green
+    physical drive '3:1-1' status: green
+    physical drive '3:1-2' status: green
+    physical drive '3:1-3' status: green
+    physical drive '3:1-4' status: green
+    physical drive '3:1-5' status: green
+    physical drive '3:1-6' status: green
+checking host 'svt02.acme.com'
+    status: alive
+    logical drive '1' status: green
+    logical drive '3' status: green
+    physical drive '1:1-1' status: green
+    physical drive '1:1-2' status: green
+    physical drive '1:1-3' status: green
+    physical drive '1:1-4' status: green
+    physical drive '1:1-5' status: green
+    physical drive '1:1-6' status: green
+    physical drive '3:1-1' status: green
+    physical drive '3:1-2' status: green
+    physical drive '3:1-3' status: green
+    physical drive '3:1-4' status: green
+    physical drive '3:1-5' status: green
+    physical drive '3:1-6' status: green
+checking host 'svt11.acme.com'
+    status: alive
+    logical drive '1' status: green
+    logical drive '3' status: green
+    physical drive '1:1-1' status: green
+    physical drive '1:1-2' status: green
+    physical drive '1:1-3' status: green
+    physical drive '1:1-4' status: green
+    physical drive '1:1-5' status: green
+    physical drive '1:1-6' status: green
+    physical drive '3:1-1' status: green
+    physical drive '3:1-2' status: green
+    physical drive '3:1-3' status: green
+    physical drive '3:1-4' status: green
+    physical drive '3:1-5' status: green
+    physical drive '3:1-6' status: green
+checking host 'svt12.acme.com'
+    status: alive
+    logical drive '1' status: green
+    logical drive '3' status: green
+    physical drive '1:1-1' status: green
+    physical drive '1:1-2' status: green
+    physical drive '1:1-3' status: green
+    physical drive '1:1-4' status: green
+    physical drive '1:1-5' status: green
+    physical drive '1:1-6' status: green
+    physical drive '3:1-1' status: green
+    physical drive '3:1-2' status: green
+    physical drive '3:1-3' status: green
+    physical drive '3:1-4' status: green
+    physical drive '3:1-5' status: green
+    physical drive '3:1-6' status: green
+```
+
```

Comment:
Missing command or at least some explanations saying thresholds are defaulted in the Plugin